bindings/py/cpp_src/bindings/algorithms/py_TemporalMemory.cpp

```diff
@@ -151,7 +151,7 @@ using namespace nupic::algorithms::connections;
 
         py_HTM.def("getPredictiveCells", [](const HTM_t& self)
         {
-            auto predictiveCells = self.getPredictiveCells();
+            auto predictiveCells = self.getPredictiveCells().getSparse();
 
             return py::array_t<nupic::UInt32>(predictiveCells.size(), predictiveCells.data());
```

Comment:
Can the python method also return an SDR, please?
- It would make the API consistent between C++ and Python, so less stuff for us to remember.
- Python also greatly benefits from using SDRs instead of lists of sparse activity.
- You should be able to just return the SDR and pybind will deal with it.

Reply:
>     ou should be able to just return the SDR and pybind will deal with it.

I didn't know it's that easy. Will do right now